app-menu/qubes-session-autostart

```diff
@@ -28,18 +28,39 @@ from qubesagent.xdg import launch, find_dropins, \
     load_desktop_entry_with_dropins
 import xdg.BaseDirectory
 import os
+import re
 
 
 QUBES_XDG_CONFIG_DROPINS = '/etc/qubes/autostart'
 
 
+def check_file_existence(path: str) -> bool:
+    if os.path.isabs(path):
+        return os.path.isfile(path)
```

Comment:
shouldn't this be `os.path.exists`? or the spec says it needs to be a normal file specifically?

Reply:
Right, it's not written that it must necessarily be a file.